packages/react-devtools-shared/src/devtools/views/root.css

```diff
@@ -1,280 +1,25 @@
-  --light-color-resize-bar: #cccccc;
-  --light-color-scheduling-profiler-native-event: #ccc;
-  --light-color-scheduling-profiler-native-event-hover: #aaa;
-  --light-color-scheduling-profiler-native-event-warning: #ee1638;
-  --light-color-scheduling-profiler-native-event-warning-hover: #da1030;
-  --light-color-scheduling-profiler-native-event-warning-text: #fff;
-  --light-color-scheduling-profiler-priority-background: #f6f6f6;
-  --light-color-scheduling-profiler-priority-border: #eeeeee;
-  --light-color-scheduling-profiler-user-timing: #c9cacd;
-  --light-color-scheduling-profiler-user-timing-hover:#93959a;
-  --light-color-scheduling-profiler-react-idle: #edf6ff;
-  --light-color-scheduling-profiler-react-idle-selected:#EDF6FF;
-  --light-color-scheduling-profiler-react-idle-hover:#EDF6FF;
-  --light-color-scheduling-profiler-react-render: #9fc3f3;
-  --light-color-scheduling-profiler-react-render-selected:#64A9F5;
-  --light-color-scheduling-profiler-react-render-hover:#2683E2;
-  --light-color-scheduling-profiler-react-commit: #ff718e;
-  --light-color-scheduling-profiler-react-commit-selected:#FF5277;
-  --light-color-scheduling-profiler-react-commit-hover:#ed0030;
-  --light-color-scheduling-profiler-react-layout-effects:#c88ff0;
-  --light-color-scheduling-profiler-react-layout-effects-selected:#934FC1;
-  --light-color-scheduling-profiler-react-layout-effects-hover:#601593;
-  --light-color-scheduling-profiler-react-passive-effects:#c88ff0;
-  --light-color-scheduling-profiler-react-passive-effects-selected:#934FC1;
-  --light-color-scheduling-profiler-react-passive-effects-hover:#601593;
-  --light-color-scheduling-profiler-react-schedule: #9fc3f3;
-  --light-color-scheduling-profiler-react-schedule-hover:#2683E2;
-  --light-color-scheduling-profiler-react-schedule-cascading:#ff718e;
-  --light-color-scheduling-profiler-react-schedule-cascading-hover:#ed0030;
-  --light-color-scheduling-profiler-react-suspend: #a6e59f;
-  --light-color-scheduling-profiler-react-suspend-hover:#13bc00;
-  --light-color-scheduling-profiler-text-color: #000000;
-  --light-color-scheduling-profiler-react-work-border:#ffffff;
   --light-color-scroll-thumb: #c2c2c2;
   --light-color-scroll-track: #fafafa;
-  --light-color-search-match: yellow;
-  --light-color-search-match-current: #f7923b;
-  --light-color-selected-tree-highlight-active: rgba(0, 136, 250, 0.1);
-  --light-color-selected-tree-highlight-inactive: rgba(0, 0, 0, 0.05);
-  --light-color-shadow: rgba(0, 0, 0, 0.25);
-  --light-color-tab-selected-border: #0088fa;
-  --light-color-text: #000000;
-  --light-color-text-invalid: #ff0000;
-  --light-color-text-selected: #ffffff;
-  --light-color-toggle-background-invalid: #fc3a4b;
-  --light-color-toggle-background-on: #0088fa;
-  --light-color-toggle-background-off: #cfd1d5;
-  --light-color-toggle-text: #ffffff;
   --light-color-tooltip-background: rgba(0, 0, 0, 0.9);
   --light-color-tooltip-text: #ffffff;
```

Comment:
I left `color-tooltip-background` and `color-tooltip-text` for this very reason
```
  // The ThemeProvider works by writing DOM style variables to an HTMLDivElement.
  // Because Portals render in a different DOM subtree, these variables don't propagate.
  // So we need to also set @reach/tooltip specific styles on the root.
```